### ISSUE TITLE
Compiling widgetset with JDK 9 don't work

### ISSUE THREAD

**Issue by mstahv** [vaadin/framework#10076]
Vaadin version: latest (also 8.2.0.alpha1 pre-release)
JDK version: latest official (JDK 9)

When you try to create a project with a widgetset, the compilation fails.

**Comment by pwilkin** [comment]
This is probably not going to be resolved until GWT releases 2.8.2 which supports JDK 9 compilation, ses https://groups.google.com/forum/#!topic/google-web-toolkit/WuoGvsjud1o

**Comment by mstahv** [comment]
Yes, I know this is GWT related, but we (Vaadin is a GWT steering committee member) should be (and should have been) active in GWT community to get the compatible version out.

**Comment by Legioth** [comment]
As mentioned in the linked discussion thread, the `HEAD-SNAPSHOT` version from https://oss.sonatype.org/content/repositories/google-snapshots/ should work perfectly fine.

We have discussed Java 9 compatibility in the GWT Steering Committee and the intention has been that GWT 2.8.2 should have been released already a couple of weeks ago. Unfortunately, things do not always go according to the plan. A release is still imminent as soon as the roadblocks have been cleared.
